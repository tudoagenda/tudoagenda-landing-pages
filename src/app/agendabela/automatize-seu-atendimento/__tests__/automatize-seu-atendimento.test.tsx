import { render } from "@testing-library/react";
import AutomatizeSeuAtendimento from "../page";
import Layout from "../layout";
import { QueryClient, QueryClientProvider } from "@tanstack/react-query";

const queryClient = new QueryClient();

describe("AutomatizeSeuAtendimento", () => {
  it("should match the snapshot for the AutomatizeSeuAtendimento component", () => {
    const { asFragment } = render(
      <QueryClientProvider client={queryClient}>
        <AutomatizeSeuAtendimento />
      </QueryClientProvider>
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it("should match the snapshot for the Layout with AutomatizeSeuAtendimento", () => {
    const { asFragment } = render(
      <Layout>
        <QueryClientProvider client={queryClient}>
          <AutomatizeSeuAtendimento />
        </QueryClientProvider>
      </Layout>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
