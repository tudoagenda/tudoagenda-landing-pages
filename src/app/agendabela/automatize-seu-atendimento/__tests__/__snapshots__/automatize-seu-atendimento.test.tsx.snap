// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutomatizeSeuAtendimento should match the snapshot for the AutomatizeSeuAtendimento component 1`] = `
<DocumentFragment>
  <main
    class="flex flex-col w-full bg-purple-900 md:p-20 p-6 items-center gap-14"
    id="teste-gratuitamente"
  >
    <img
      alt="AgendaBela logo"
      data-nimg="1"
      decoding="async"
      height="97"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
      style="color: transparent;"
      width="170"
    />
    <div
      class="flex flex-col text-center max-w-4xl items-center gap-4"
    >
      <h1
        class="md:text-5xl text-white text-4xl max-w-3xl font-bold"
      >
        Automatize seu atendimento sem dores de cabeça.
      </h1>
      <div
        class="flex flex-col text-purple-300 md:text-xl text-sm font-lexend"
      >
        <p>
          Nós somos experts em tecnologia. Você não precisa ser.
        </p>
        <p>
          Criamos o AgendaBela pra você automatizar seu atendimento e gerenciar melhor seu salão.
        </p>
      </div>
    </div>
    <div
      class="flex flex-col gap-4 w-full max-w-2xl items-center"
    >
      <form
        class="flex gap-2 w-full max-w-lg"
      >
        <input
          class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-base ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
          name="email"
          placeholder="Digite seu e-mail aqui"
          required=""
          type="email"
        />
        <button
          class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-10 px-4 py-2"
          type="submit"
        >
          Teste Gratuitamente
        </button>
      </form>
      <div
        class="md:text-xs text-[11px] font-lexend text-purple-500 text-center"
      >
        <p>
          Teste nossa plataforma por 30 dias gratuitamente. 
          <u>
            Não vamos te pedir nenhum dado de cartão de crédito agora.
          </u>
        </p>
      </div>
    </div>
  </main>
  <article
    class="flex flex-col w-full md:p-12 p-6 items-center gap-10 md:gap-20"
  >
    <h2
      class="text-purple-950 font-bold text-3xl md:text-4xl text-center md:text-left"
    >
      Saia na frente dos outros salões em 3 passos:
    </h2>
    <section
      class="flex flex-col md:flex-row w-full max-w-7xl justify-between gap-8"
    >
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-store"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m2 7 4.41-4.41A2 2 0 0 1 7.83 2h8.34a2 2 0 0 1 1.42.59L22 7"
          />
          <path
            d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"
          />
          <path
            d="M15 22v-4a2 2 0 0 0-2-2h-2a2 2 0 0 0-2 2v4"
          />
          <path
            d="M2 7h20"
          />
          <path
            d="M22 7v3a2 2 0 0 1-2 2a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 16 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 12 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 8 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 4 12a2 2 0 0 1-2-2V7"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          1 Configure seu salão
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Talvez você já conheceu outras plataformas onde, pra configurar o seu salão, era quase que um exercício de matemática. O AgendaBela não é assim. Simples e rápido, com tutoriais em vídeo, você vai conseguir configurar o seu salão em 5 minutos!
        </p>
      </div>
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-smartphone"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            height="20"
            rx="2"
            ry="2"
            width="14"
            x="5"
            y="2"
          />
          <path
            d="M12 18h.01"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          2 Peça seu atendente virtual
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Sim, nós vamos criar um atendente virtual para você no WhatsApp. Mas não se preocupe, você não vai perder o seu número atual. Nós criamos um novo número para o seu atendente virtual. Seus clientes podem interagir com ele e, com poucos cliques, serem atendidos rapidamente.
        </p>
      </div>
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-users-round"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M18 21a8 8 0 0 0-16 0"
          />
          <circle
            cx="10"
            cy="8"
            r="5"
          />
          <path
            d="M22 20c0-3.37-2-6.5-4-8a5 5 0 0 0-.45-8.3"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          3 Agora é só falar para seus clientes
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Configure uma mensagem automática no seu WhatsApp atual para redirecionar seus clientes para serem atendidos pelo atendente virtual. Nós te ajudamos com isso também, não se preocupe. Pronto! É só isso mesmo. Seu salão já está na frente de 90% dos outros salões.
        </p>
      </div>
    </section>
    <a
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-11 rounded-md px-8"
      href="#teste-gratuitamente"
    >
      Teste Gratuitamente
    </a>
  </article>
  <div
    class="flex flex-col items-center w-full px-8 md:px-16 py-16"
  >
    <div
      class="flex gap-8 p-12 w-full max-w-7xl border-y border-y-purple-200 items-center justify-center"
    >
      <img
        alt="Santander X logo"
        data-nimg="1"
        decoding="async"
        height="27"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
        style="color: transparent;"
        width="200"
      />
      <img
        alt="Founders Club logo"
        data-nimg="1"
        decoding="async"
        height="80"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
        style="color: transparent;"
        width="141"
      />
    </div>
  </div>
  <div
    class="flex flex-col w-full p-10 gap-16 items-center"
  >
    <div
      class="flex flex-col w-full max-w-3xl text-center gap-4 items-center"
    >
      <p
        class="text-2xl text-purple-950"
      >
        ”O Agenda Bela facilitou muito o dia-a-dia aqui no salão. Agora eu consigo gerenciar a minha agenda e ter uma visão muito melhor do meu trabalho por aqui”
      </p>
      <p
        class="text-lg font-semibold text-purple-950"
      >
        MARIA JOANA, PROPRIETÁRIA DO SALÃO PARIS
      </p>
    </div>
    <a
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-11 rounded-md px-8"
      href="#teste-gratuitamente"
    >
      Teste Gratuitamente
    </a>
  </div>
  <footer
    class="flex w-full p-8 justify-center"
  >
    <button
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-primary underline-offset-4 hover:underline h-10 px-4 py-2"
    >
      Termos de serviço
    </button>
    <button
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-primary underline-offset-4 hover:underline h-10 px-4 py-2"
    >
      Política de privacidade
    </button>
  </footer>
</DocumentFragment>
`;

exports[`AutomatizeSeuAtendimento should match the snapshot for the Layout with AutomatizeSeuAtendimento 1`] = `
<DocumentFragment>
  <main
    class="flex flex-col w-full bg-purple-900 md:p-20 p-6 items-center gap-14"
    id="teste-gratuitamente"
  >
    <img
      alt="AgendaBela logo"
      data-nimg="1"
      decoding="async"
      height="97"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
      style="color: transparent;"
      width="170"
    />
    <div
      class="flex flex-col text-center max-w-4xl items-center gap-4"
    >
      <h1
        class="md:text-5xl text-white text-4xl max-w-3xl font-bold"
      >
        Automatize seu atendimento sem dores de cabeça.
      </h1>
      <div
        class="flex flex-col text-purple-300 md:text-xl text-sm font-lexend"
      >
        <p>
          Nós somos experts em tecnologia. Você não precisa ser.
        </p>
        <p>
          Criamos o AgendaBela pra você automatizar seu atendimento e gerenciar melhor seu salão.
        </p>
      </div>
    </div>
    <div
      class="flex flex-col gap-4 w-full max-w-2xl items-center"
    >
      <form
        class="flex gap-2 w-full max-w-lg"
      >
        <input
          class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-base ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
          name="email"
          placeholder="Digite seu e-mail aqui"
          required=""
          type="email"
        />
        <button
          class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-10 px-4 py-2"
          type="submit"
        >
          Teste Gratuitamente
        </button>
      </form>
      <div
        class="md:text-xs text-[11px] font-lexend text-purple-500 text-center"
      >
        <p>
          Teste nossa plataforma por 30 dias gratuitamente. 
          <u>
            Não vamos te pedir nenhum dado de cartão de crédito agora.
          </u>
        </p>
      </div>
    </div>
  </main>
  <article
    class="flex flex-col w-full md:p-12 p-6 items-center gap-10 md:gap-20"
  >
    <h2
      class="text-purple-950 font-bold text-3xl md:text-4xl text-center md:text-left"
    >
      Saia na frente dos outros salões em 3 passos:
    </h2>
    <section
      class="flex flex-col md:flex-row w-full max-w-7xl justify-between gap-8"
    >
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-store"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m2 7 4.41-4.41A2 2 0 0 1 7.83 2h8.34a2 2 0 0 1 1.42.59L22 7"
          />
          <path
            d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"
          />
          <path
            d="M15 22v-4a2 2 0 0 0-2-2h-2a2 2 0 0 0-2 2v4"
          />
          <path
            d="M2 7h20"
          />
          <path
            d="M22 7v3a2 2 0 0 1-2 2a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 16 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 12 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 8 12a2.7 2.7 0 0 1-1.59-.63.7.7 0 0 0-.82 0A2.7 2.7 0 0 1 4 12a2 2 0 0 1-2-2V7"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          1 Configure seu salão
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Talvez você já conheceu outras plataformas onde, pra configurar o seu salão, era quase que um exercício de matemática. O AgendaBela não é assim. Simples e rápido, com tutoriais em vídeo, você vai conseguir configurar o seu salão em 5 minutos!
        </p>
      </div>
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-smartphone"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            height="20"
            rx="2"
            ry="2"
            width="14"
            x="5"
            y="2"
          />
          <path
            d="M12 18h.01"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          2 Peça seu atendente virtual
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Sim, nós vamos criar um atendente virtual para você no WhatsApp. Mas não se preocupe, você não vai perder o seu número atual. Nós criamos um novo número para o seu atendente virtual. Seus clientes podem interagir com ele e, com poucos cliques, serem atendidos rapidamente.
        </p>
      </div>
      <div
        class="flex flex-col w-full max-w-sm gap-2"
      >
        <svg
          class="lucide lucide-users-round"
          fill="none"
          height="32"
          stroke="#A855F7"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M18 21a8 8 0 0 0-16 0"
          />
          <circle
            cx="10"
            cy="8"
            r="5"
          />
          <path
            d="M22 20c0-3.37-2-6.5-4-8a5 5 0 0 0-.45-8.3"
          />
        </svg>
        <h3
          class="text-purple-950 font-bold text-lg"
        >
          3 Agora é só falar para seus clientes
        </h3>
        <p
          class="text-purple-950 text-base"
        >
          Configure uma mensagem automática no seu WhatsApp atual para redirecionar seus clientes para serem atendidos pelo atendente virtual. Nós te ajudamos com isso também, não se preocupe. Pronto! É só isso mesmo. Seu salão já está na frente de 90% dos outros salões.
        </p>
      </div>
    </section>
    <a
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-11 rounded-md px-8"
      href="#teste-gratuitamente"
    >
      Teste Gratuitamente
    </a>
  </article>
  <div
    class="flex flex-col items-center w-full px-8 md:px-16 py-16"
  >
    <div
      class="flex gap-8 p-12 w-full max-w-7xl border-y border-y-purple-200 items-center justify-center"
    >
      <img
        alt="Santander X logo"
        data-nimg="1"
        decoding="async"
        height="27"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
        style="color: transparent;"
        width="200"
      />
      <img
        alt="Founders Club logo"
        data-nimg="1"
        decoding="async"
        height="80"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
        style="color: transparent;"
        width="141"
      />
    </div>
  </div>
  <div
    class="flex flex-col w-full p-10 gap-16 items-center"
  >
    <div
      class="flex flex-col w-full max-w-3xl text-center gap-4 items-center"
    >
      <p
        class="text-2xl text-purple-950"
      >
        ”O Agenda Bela facilitou muito o dia-a-dia aqui no salão. Agora eu consigo gerenciar a minha agenda e ter uma visão muito melhor do meu trabalho por aqui”
      </p>
      <p
        class="text-lg font-semibold text-purple-950"
      >
        MARIA JOANA, PROPRIETÁRIA DO SALÃO PARIS
      </p>
    </div>
    <a
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-black bg-amber-500 hover:bg-amber-600 h-11 rounded-md px-8"
      href="#teste-gratuitamente"
    >
      Teste Gratuitamente
    </a>
  </div>
  <footer
    class="flex w-full p-8 justify-center"
  >
    <button
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-primary underline-offset-4 hover:underline h-10 px-4 py-2"
    >
      Termos de serviço
    </button>
    <button
      class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 text-primary underline-offset-4 hover:underline h-10 px-4 py-2"
    >
      Política de privacidade
    </button>
  </footer>
</DocumentFragment>
`;
